components/payments/cmd/connectors/internal/connectors/atlar/DEVELOPERS.md

```diff
@@ -0,0 +1,9 @@
+# Atlar HTTP Client
+
+The directories `client` and `models` have been automatically generated by means of running
+
+```sh
+docker run --rm -it  --user $(id -u):$(id -g) -e GOPATH=$(go env GOPATH):/go -v $HOME:$HOME -w $(pwd) quay.io/goswagger/swagger generate client -f https://cdn.atlar.com/api/swagger.json -A client
```

Comment:
No strong opinion here, but we're not fans of putting 250 files for the client part inside this repository, could it be possible to put them in another public repository while waiting for them to handle a go library on their sides ? 

What do you think ?

Reply:
At least for `v1` Atlar is not going to provide a go library. Here's their response for reference:

<img width="791" alt="Screenshot 2023-11-29 at 15 31 46" src="https://github.com/formancehq/stack/assets/91877506/d6588552-1db3-4ebc-812b-8751dfb2e0cd">

I'll ask Sonny about putting a standalone Atlar `v1` client up on a GH repo. We'll have to figure out the licensing. I can start looking into that.

Reply:
Atlar client is an external package, now.

Reply:
Perfect thank you !